src/main/java/billgates/view/gui/ActionPanel.java

```diff
@@ -215,7 +217,48 @@ private void backFromSplit() {
     }
 
     private void addEntry() {
-        // TODO: Call the controller of InsertEntryUseCase
+        JTextField dateField = new JTextField(20);
+        JTextField valueField = new JTextField(20);
+        JTextField currencyField = new JTextField(20);
+        JTextField descriptionField = new JTextField(20);
+        JTextField fromField = new JTextField(20);
+        JTextField toField = new JTextField(20);
+        JTextField locationField = new JTextField(20);
+
+        JPanel myPanel = new JPanel();
+        myPanel.setLayout(new BoxLayout(myPanel, BoxLayout.PAGE_AXIS));
+        myPanel.add(new JLabel("date:"));
+        myPanel.add(dateField);
+        myPanel.add(new JLabel("value:"));
+        myPanel.add(valueField);
+        myPanel.add(new JLabel("currency:"));
+        myPanel.add(currencyField);
+        myPanel.add(new JLabel("description:"));
+        myPanel.add(descriptionField);
+        myPanel.add(new JLabel("from:"));
+        myPanel.add(fromField);
+        myPanel.add(new JLabel("to:"));
+        myPanel.add(toField);
+        myPanel.add(new JLabel("location:"));
+        myPanel.add(locationField);
+
+        int result = JOptionPane.showConfirmDialog(null, myPanel,
+                "Information about the new entry", JOptionPane.OK_CANCEL_OPTION);
+        if (result == JOptionPane.OK_OPTION) {
+            String date1 = dateField.getText();
+            ZonedDateTime date = ZonedDateTime.parse(date1);
```

Comment:
This cases some exceptions with formatting datetime.

The solution could be replacing this line with 
```java
LocalDateTime localDateTime = LocalDateTime.parse(date1, DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss"));
ZonedDateTime date = ZonedDateTime.of(localDateTime, ZoneId.systemDefault());
```